test/util/ChartUtils/getStackedData.spec.ts

```diff
@@ -0,0 +1,284 @@
+import { Series, SeriesPoint } from 'victory-vendor/d3-shape';
+import { getStackedData, offsetSign } from '../../../src/util/ChartUtils';
+import { DataKey, StackOffsetType } from '../../../src/util/types';
+
+function dk(dataKey: DataKey<any>) {
+  return { props: { dataKey } };
+}
+
+function s(
+  key: string,
+  index: number,
+  data: Array<SeriesPoint<Record<string, unknown>>> = [],
+): Series<Record<string, unknown>, string> {
+  return Object.assign(data, { key, index });
+}
+
+function sp(first: number, second: number, data: Record<string, unknown>): SeriesPoint<Record<string, unknown>> {
+  return Object.assign([], { data, 0: first, 1: second });
+}
```

Comment:
Unless there exists a convention that I am ignorant of, please do provide proper names to these functions. Tests must also be maintained, and using abbreviations does not make it easier.